draft-iyengar-quic-delayed-ack.md

```diff
@@ -349,6 +327,41 @@ flight at any given time, this extension does not prohibit having more than one
 in flight. Generally, when using `max_ack_delay` for PTO computations, endpoints
 MUST use the maximum of the current value and all those in flight.
 
+# Implementation Considerations {#implementation}
+
+There are tradeoffs inherent in a sender sending an ACK_FREQUENCY frame to the
+receiver. As such it is recommended that implementers experiment with different
+strategies and find those which best suit their applications and networks.
+There are, however, noteworthy considerations when devising strategies for
+sending ACK_FREQUENCY frames.
+
+## Loss Detection {#loss}
+A sender relies on receipt of acknowledgements to determine the amount of data
+in flight and to detect losses, see {{QUIC-RECOVERY}}. Consequently, how often a
+receiver sends acknowledgments determines how long it takes for losses to be
+detected at the sender.
+
+## Nascent Connections {#nascent}
+Many congestion control algorithms have a special state for the begginning
+phases of a connection.  It is typical that in this phase the congestion
+controller will gradually increase the amount of data in the network until it is
+signalled to stop.  While the mechanism used to achieve this increase varies,
+acknowledgments by the peer are generally critical during this phase to drive
+the congestion controller's machinery.  Hence, it is advisable that caution is
+exercised in changing the receiver's acknowledgment behavior while the sender's
+congestion controller is in this state.  Delaying acknowledgments excessively
+can lead to increased delay in the sender's congestion controller reaching an
+optimal value, leading to poor performance.
```

Comment:
This section seems a bit odd down here given this draft adds an explicit signal for the sender to tell the receiver what ACK strategy it wants.  The section could be re-written to advise the sender to delay sending the ACK_FREQUENCY frame while it's reliant upon frequent acknowledgements to quickly ramp up the congestion controller, and remove the part about how the receiver needs to be cautious about changing its strategy.

Reply:
I think I phrased this poorly. What I meant is that the _sender_ should be cautious in sending an ACK_FREQUENCY too early in the connection, as indeed the receiver is meant to be a passive actor in this decision making. I will rephrase.